changelog/1.23.0.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "1.23.0"
+description: "Released on 09/22/2021"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.23.0.
+
+### Features ✨
+
+- Doctor: [Doctor](https://github.com/cdr/coder-doctor) assesses a Kubernetes
+  cluster to determine its readiness for Coder installation; if there are
+  issues, Doctor provides information on why.
+- Terraform: Coder has released the first of a series of
+  [Terraform scripts](https://github.com/cdr/enterprise-terraform) to facilitate
+  one-click Coder deployment.
+
```

Comment:
```suggestion
```

I think this newline is causing the markdown renderer to insert extra paragraph elements:
![image](https://user-images.githubusercontent.com/4040560/134379686-c18589b5-3b75-489f-81c9-10d08da2573b.png)